README.md

```diff
@@ -113,6 +113,13 @@ For convenience, you can also set the
 put the `.clang` file or the compilation database. It overrides the
 default search directory.
 
+If preferred, you can set the `g:chromatica#search_source_args` option to
+have Chromatica the compilation database for similar filenames, if an entry
```

Comment:
You forgot a verb between “Chromatica” and “the compilation database”.

Reply:
Ah, you're totally right, whoops!
```suggestion
have Chromatica search the compilation database for similar filenames, if an entry
```